force-app/main/default/classes/RD2_ChangeHistoryService_TEST.cls

```diff
@@ -77,6 +78,12 @@ public with sharing class RD2_ChangeHistoryService_TEST {
 
         rdchList = getRecurringDonationChangeHistory(rd.Id);
         System.assertEquals(0, rdchList.size(), 'There should be no Change History with the Setting not Enabled');
+
+        // Make sure that the Change Type is cleared, even if it wasn't used
+        Id rdId = rd.Id;
+        List<npe03__Recurring_Donation__c> rdList = 
+            [SELECT Id, ChangeType__c FROM npe03__Recurring_Donation__c WHERE Id = :rdId];
```

Comment:
We have a rdGatewat class that is used in Test method to query RD. `private static final TEST_SObjectGateway.RecurringDonationGateway rdGateway = new TEST_SObjectGateway.RecurringDonationGateway();` Using this class can reduce the amount of RD query being written in this test class

Reply:
Great reminder, I just updated them.